### ISSUE TITLE
How to set the half_pixel_centers of ResizeBilinear to False in the .tflite?

### ISSUE THREAD

**Issue by zhanggd001** [tensorflow/tensorflow#58684]
### 1. System information

- OS Platform and Distribution (Linux Ubuntu 22.04):

Hi, i convert the model from pytorch into tflite：

   pytorch = 1.10 (.pth) -> onnx = 1.12 (.onnx) -> tensorflow = 2.11  (.tflite)

I find the half_pixel_centers  of the ResizeBilinear  operator is True, but i need this option to be False.  

How can i set the half_pixel_centers to False during model convert?

![image](https://user-images.githubusercontent.com/48872511/203933849-b9b0564b-85ec-427e-a2b0-ab32c1413c1a.png)

I try to set the half_pixel_centers  to False, when parsing the onnx model, and converting it into keras model:

---------------------------------------------------------------------------------------------------------------------------------
def keras_builder(onnx_model, new_input_nodes:list=None, new_output_nodes:list=None):
    model_graph = onnx_model.graph

    '''
        init onnx model's build-in tensors
    '''
    onnx_weights = dict()
    for initializer in model_graph.initializer:
        onnx_weights[initializer.name] = numpy_helper.to_array(initializer)

    '''
        build input nodes
    '''
    tf_tensor, input_shape = {}, []
    for inp in model_graph.input:
        input_shape = [x.dim_value for x in inp.type.tensor_type.shape.dim]
        if input_shape == []:
            continue
        batch_size = 1 if input_shape[0] <= 0 else input_shape[0]
        input_shape = input_shape[2:] + input_shape[1:2]
        tf_tensor[inp.name] = keras.Input(shape=input_shape, batch_size=batch_size)

    '''
        build model inline node by iterate onnx nodes.
    '''
    input_node_names, outputs_node_names = [], []
    for node in model_graph.node:
        op_name, node_inputs, node_outputs, node_name = node.op_type, node.input, node.output, node.name
        op_attr = decode_node_attribute(node)
        
        #print("opt name {}, opt {}".format(op_name, op_attr))

        tf_operator = OPERATOR.get(op_name)

        if(op_name == "Resize"):
            print(tf_operator)
            op_attr['pytorch_half_pixel'] = False

        if tf_operator is None:
            raise KeyError(f"{op_name} not implemented yet")
        
        _inputs = None 
        if len(node_inputs) > 0:
            _inputs = tf_tensor[node_inputs[0]] if node_inputs[0] in tf_tensor else onnx_weights[node_inputs[0]]

        for index in range(len(node_outputs)):
            tf_tensor[node_outputs[index]] = tf_operator(tf_tensor, onnx_weights, node_inputs, op_attr, index=index)(_inputs)
            if(op_name == "Resize"):
                print(tf_operator(tf_tensor, onnx_weights, node_inputs, op_attr, index=index)(_inputs))
        '''
            reorganize input and output nodes
        '''
        if new_input_nodes is not None and node_name in new_input_nodes:
            input_node_names.append(node_outputs[0])
        # TODO for nodes with multiply outputs.
        if new_output_nodes is not None and node_name in new_output_nodes:
            outputs_node_names.append(node_outputs[0])
        if new_output_nodes is not None and len(outputs_node_names) == len(new_output_nodes):
            break
    
    '''
        process input and output nodes 
    '''
    input_nodes = []
    if new_input_nodes is None:
        input_nodes = [tf_tensor[x.name] for x in model_graph.input]
    else:
        for node in model_graph.input:
            if node.name in new_input_nodes:
                input_node_names.append(node.name)
        input_nodes = [tf_tensor[x] for x in input_node_names]
    outputs_nodes = []
    if new_output_nodes is None:
        outputs_nodes = [tf_tensor[x.name] for x in model_graph.output]
    else:
        for node in model_graph.output:
            if node.name in new_output_nodes:
                outputs_node_names.append(node.name)
        outputs_nodes = [tf_tensor[x] for x in outputs_node_names]

    '''
        build keras model
    '''
    keras_model = keras.Model(inputs=input_nodes, outputs=outputs_nodes)
    keras_model.trainable = False
    # keras_model.summary()

    return keras_model
------------------------------------------------------------------------------------------------------------------
    keras_model = keras_builder(model_proto, input_node_names, output_node_names)
    print("Build keras model Done")
    # set resize half_pixel_centers = False
    ind =0
    for layer in keras_model.layers:

        #print(layer.name)

        if ("resize" in layer.name) or ("Resize" in layer.name):
            print("###############################")
            
            keras_model.layers[ind].half_pixel_centers = False
            print(keras_model.layers[ind].half_pixel_centers)
         
        ind+=1

But when converting keras to tflite model by tf.lite.TFLiteConverter.from_keras_model(keras_model),  the half_pixel_centers  is still True.

**Comment by mohantym** [comment]
Hi @zhanggd001 !

You can use [tfl.quantization_debugger ](https://www.tensorflow.org/lite/performance/quantization_debugger)to disable the layers. But not sure on setting half_pixel_centers to False through that. Feel free to see [tensorflow model optimization ](https://www.tensorflow.org/model_optimization)documentation too. 

Thank you!
